README.md

```diff
@@ -1,19 +1,54 @@
 # opencast-bigbluebutton-recordings-plugin
 
+This plugin serves BigBlueButton frontends recordings from an Opencast instead of the actual
+BigBlueButton. It does this by redirecting calls to the BBB-API `/bigbluebutton/api/getRecordings`
+to the Opencast API. This allows to create recordings BBB recordings to be hosted in Opencast
+and have them still show up in BBB frontends, without requiring any changes to the said
+frontends. 
+
+To successfully retrieve a recording from Opencast, the recordings UUID needs to match the
+meetingID specified in the query.
+
+For more information on how to get your BBB recordings to Opencast, visit: 
+https://github.com/elan-ev/opencast-bigbluebutton-integration/tree/master/post-publish
+
 ## Build
 
+To install the plugin, clone the repository onto your BigBlueButton-Server 
+and run the commands below. You will need [Golang](https://golang.org/) to build successfully.
+
 ```
 go mod download
 go build
 ```
 
+Note: It is recommended to install Golang from an official archive at https://golang.org/doc/install,
+as the current debian package does not include the `mod` command.
```

Comment:
If you built on the Big Blue Button Server, it was properly not Debian, but Ubuntu 16.04.7 LTS, but as mentioned earlier, better not build on the production server anyway.